### DISCUSSION TITLE
Merge Conflicts

### DISCUSSION THREAD

**Discussion by learnwithums** [learnwithums/test1#14] [Q&A] [answered]
How do I resolve merge Conflicts?

**Comment by ums91** [comment, marked as answer]
> How do I resolve merge Conflicts?

Merge conflicts occur when there are competing changes in the same file or line. To resolve them:
Use git pull to merge the latest changes from the target branch.
Edit the files with conflicts, following Git’s markers (e.g., <<<<<<< HEAD).
Once resolved, stage the changes, commit, and push

**Reply by learnwithums** [reply]
thanks
